### PR TITLE
Update zoom / pan bounds based on offset from center

### PR DESCRIPTION
**Pull request by cathyxz**
Closes https://github.com/ampproject/amphtml/issues/17417. 
This probably needs multiple device testing.

### CODE REVIEW

File: extensions/amp-pan-zoom/0.1/amp-pan-zoom.js [hunk 1]
```diff
@@ -303,12 +309,20 @@ export class AmpPanZoom extends AMP.BaseElement {
...
     this.contentBox_ = layoutRectLtwh(
-        Math.round((this.elementBox_.width - width) / 2),
-        Math.round((this.elementBox_.height - height) / 2),
+        contentBox.left - this.elementBox_.left,
+        contentBox.top - this.elementBox_.top,
         Math.round(width),
         Math.round(height));
 
+    this.yOffsetFromCenter_ = this.contentBox_.top
+      - (this.elementBox_.height - this.contentBox_.height) / 2;
```
Comment (aghassemi) [inline extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: lint didn't complain about this?! 😮 :)

File: extensions/amp-pan-zoom/0.1/amp-pan-zoom.js [hunk 2]
```diff
@@ -523,15 +537,19 @@ export class AmpPanZoom extends AMP.BaseElement {
     const dh = this.elementBox_.height - (this.contentBox_.height * scale);
     const dw = this.elementBox_.width - (this.contentBox_.width * scale);
 
-    const minY = dh >= 0 ? 0 : dh / 2;
-    const maxY = dh >= 0 ? 0 : -minY;
+    const minY = dh >= 0 ? 0 : dh / 2 ;
```
Comment (aghassemi) [inline extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: nit: space

File: extensions/amp-pan-zoom/0.1/amp-pan-zoom.js (hunk 1 above)
Comment (cathyxz) [reply extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: Lol. Let me see what I can do about making this more comprehensible for people who are similarly bad at math as myself.

**Comment by aghassemi** [comment]
LGTM

**Pull request merged**